docs/users-guide/dashboard-subscriptions.md

```diff
@@ -0,0 +1,50 @@
+## Dashboard subscriptions
+
+Dashboard subscriptions are a great way to keep you and your team up to date on the data that matters most. They allow you to send all of the questions on a dashboard via email or Slack. If your Metabase has email or Slack set up, all you need to do is create a dashboard, add subscribers to it, and tell Metabase how often you'd like the send out an update. You can set up as many subscriptions to a dashboard as you like, and if you make any changes to the dashboard, Metabase will update the subscription the next time they're delivered.
+
+### Enabling dashboard subscriptions
+
+To enable dashboard subscriptions, your administrators will need to have set up email or Slack for your Metabase. See [Setting up email](https://www.metabase.com/docs/latest/administration-guide/02-setting-up-email.html) or [Setting up Slack](https://www.metabase.com/docs/latest/administration-guide/09-setting-up-slack.html).
+
+### Setting up a dashboard subscription
+
+To set up a subscription to a dashboard, click on the **sharing** icon (the one-way arrow) and select **Dashboard subscriptions**.
+
+![Select dashboard subscriptions](images/dashboard-subscriptions/select-dashboard-subscription.png)
+
+Metabase will slide out a sidebar on the right, with an option set up a subscription via email or Slack:
+
+![Set up a dashboard subscription with email or slack](images/dashboard-subscriptions/email-or-slack.png)
+
+Let's say we want to email a dashboard. We'll click on the **Email it** option in the sidebar, and Metabase will give us some options:
+
+![Dashboard subscription email options](images/dashboard-subscriptions/email-options.png)
+
+### Email subscription options
+
+For emails, we can:
```

Comment:
I'd advocate for listing the options in the order they appear. I think the send-now button's bullet and the attach results bullet are in the wrong positions.